README.rst

```diff
@@ -59,7 +59,7 @@ Unsupported Python Versions
 Python == 2.7, Python == 3.5, Python == 3.6.
 
 The last version of this library compatible with Python 2.7 and 3.5 is
-`google-cloud-bigquery==1.28.0`.
+`bigquery-magics==0.1.0`.
 
```

Comment:
Not true. We don't have any version with Python 2.7 support. Please remove these lines.